.github/workflows/main.yml

```diff
@@ -102,9 +102,14 @@ jobs:
   update-semver:
     if: startsWith(github.ref, 'refs/tags/v')
     needs: build_action
-    permissions:
-      contents: write
```

Comment:
Why do we remove these lines?

Reply:
In this case we have global permissions

https://github.com/splunk/appinspect-cli-action/blob/3ec92566710667f92335b421aa733232e660b32a/.github/workflows/main.yml#L28